docs/tutorials/gymnasium_basics/environment_creation.py

```diff
@@ -5,83 +5,46 @@
 
 This documentation overviews creating new environments and relevant
 useful wrappers, utilities and tests included in Gymnasium designed for
-the creation of new environments.
-
-
-Setup
-------
-
-Recommended solution
-~~~~~~~~~~~~~~~~~~~~
-
-1. Install ``pipx`` following the `pipx documentation <https://pypa.github.io/pipx/installation/>`_.
-2. Then install Copier:
-
-.. code:: console
-
-    pipx install copier
-
-Alternative solutions
-~~~~~~~~~~~~~~~~~~~~
-
-Install Copier with Pip or Conda:
-
-.. code:: console
-
-    pip install copier
-
-or
-
-.. code:: console
-
-    conda install -c conda-forge copier
-
-
-Generate your environment
-------------------------------
-
-You can check that ``Copier`` has been correctly installed by running the following command, which should output a version number:
-
-.. code:: console
-
-    copier --version
-
-Then you can just run the following command and replace the string ``path/to/directory`` by the path to the directory where you want to create your new project.
+the creation of new environments. You can clone gym-examples to play
+with the code that is presented here. We recommend that you use a virtual environment:
 
 .. code:: console
 
-    copier copy https://github.com/Farama-Foundation/gymnasium-env-template.git "path/to/directory"
-
-Answer the questions, and when it's finished you should get a project structure like the following:
-
-.. code:: sh
-
-    .
-    ├── gymnasium_env
-    │   ├── envs
-    │   │   ├── grid_world.py
-    │   │   └── __init__.py
-    │   ├── __init__.py
-    │   └── wrappers
-    │       ├── clip_reward.py
-    │       ├── discrete_actions.py
-    │       ├── __init__.py
-    │       ├── reacher_weighted_reward.py
-    │       └── relative_position.py
-    ├── LICENSE
-    ├── pyproject.toml
-    └── README.md
+   git clone https://github.com/Farama-Foundation/gym-examples
+   cd gym-examples
+   python -m venv .env
```

Comment:
Just a small nitpicky thing, but I think another name for the venv should be used since .env is usualy a txt file meant for environment variables in the context of JS.